packages/dds/merge-tree/src/mergeTree.ts

```diff
@@ -1562,29 +1562,47 @@ export class MergeTree {
 				backwardExcursion(newSegment, findLeftMovedSegment);
 				forwardExcursion(newSegment, findRightMovedSegment);
 
-				let found: ObliterateInfo | undefined;
+				let oldest: ObliterateInfo | undefined;
+				let normalizedOldestSeq: number = 0;
+				let newest: ObliterateInfo | undefined;
+				let normalizedNewestSeq: number = 0;
 				const movedClientIds: number[] = [];
 				const movedSeqs: number[] = [];
 				for (const ob of overlapping) {
-					if (ob.seq === UnassignedSequenceNumber || ob.seq > refSeq) {
-						if (found === undefined || found.seq < ob.seq) {
-							found = ob;
+					// compute a normalized seq that takes into account local seqs
+					// but is still comparable to remote seqs to keep the checks below easy
+					// REMOTE SEQUENCE NUMBERS                                     LOCAL SEQUENCE NUMBERS
+					// [0, 1, 2, 3, ..., 100, ..., 1000, ..., (MAX - MaxLocalSeq), L1, L2, L3, L4, ..., L100, ..., L1000, ...(MAX)]
+					const normalizedObSeq =
+						ob.seq === UnassignedSequenceNumber
+							? Number.MAX_SAFE_INTEGER - this.collabWindow.localSeq + ob.localSeq!
+							: ob.seq;
+					if (normalizedObSeq > refSeq) {
+						if (oldest === undefined || normalizedOldestSeq > normalizedObSeq) {
+							normalizedOldestSeq = normalizedObSeq;
+							oldest = ob;
 							movedClientIds.unshift(ob.clientId);
 							movedSeqs.unshift(ob.seq);
+						}
+						if (newest === undefined && normalizedNewestSeq < normalizedObSeq) {
+							normalizedNewestSeq = normalizedObSeq;
+							newest = ob;
+							movedClientIds.push(ob.clientId);
+							movedSeqs.push(ob.seq);
 						} else {
 							movedClientIds.push(ob.clientId);
 							movedSeqs.push(ob.seq);
 						}
 					}
 				}
 
-				if (found) {
+				if (oldest && newest?.clientId !== clientId) {
```

Comment:
we use the newest obliterate to determine if the client can bypass the obliterate, but we use the data from the oldest obliterate which matches existing behavior